tmeister/static/src/root.routes.js

```diff
@@ -3,18 +3,28 @@ import { Router, Route, Link, Redirect, hashHistory } from 'react-router';
 import TogglesContainer from './toggles/toggles.container.js';
 import RootContainer from './menu/root.container.js';
 import AuditTrail from './audit-trail/audit-trail.container.js';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { ThemeProvider } from '@material-ui/styles'
+
+const palette = {
+  primary: { main: '#00BF4B' },
+  secondary: { main: '#3399ff' }
+};
```

Comment:
Using our colors for the theme

Reply:
You should probably add a contrast text color:
```
primary: {
  main: '#00BF4B',
  contrastText: '#ffffff',
},
```